Formula/openmittsu.rb

```diff
@@ -0,0 +1,33 @@
+class Openmittsu < Formula
+  desc "Cross-platform GPLv2 desktop client for the Threema Messenger App"
+  homepage "https://github.com/blizzard4591/openMittsu"
+  url "https://github.com/blizzard4591/openMittsu/archive/0.9.15.tar.gz"
+  sha256 "7c70de775716f8c01c6f6f4c812fd938f2aea979f6fe4562bb6a4089fa3b9cae"
+  head "https://github.com/blizzard4591/openMittsu.git", :using => :git, :shallow => false
+
+  depends_on "blizzard4591/homebrew-qt5-sqlcipher/qt5-sqlcipher"
+  depends_on "cmake"
+  depends_on "libsodium"
+  depends_on :macos => :mavericks
+  depends_on "pkg-config"
```

Comment:
Is this required at runtime?

Reply:
No, only at build-time. How would I specify this?

Reply:
https://docs.brew.sh/Formula-Cookbook#specifying-other-formulae-as-dependencies